ui/src/components/policy/PolicyList.js

```diff
@@ -95,13 +150,48 @@ export default class PolicyList extends React.Component {
             });
     }
 
+    onSubmitDuplicatePolicy() {
+        this.api
+            .duplicatePolicyVersion(
+                this.props.domain,
+                this.state.duplicatePolicyName,
+                this.state.duplicateVersionSourceName,
+                this.state.duplicateVersionName,
+                this.props._csrf
+            )
+            .then(() => {
+                this.state.reloadPolicyVersionsFunc(
+                    'duplicatePolicyVersion',
```

Comment:
Call reloadPolicyVersionsFunc to expand the policy versions, update state and eventually call closeModals to close the dialog modals.